src/main/webapp/service-worker.js

```diff
@@ -0,0 +1,56 @@
+const CACHE_VERSION = 1;
```

Comment:
Yes, CACHE_VERSION is just part of the name of the cache and is not changed anywhere, and since the only thing we're caching is census data there will only be one cache in CURRENT_CACHES. These lines and the 'activate' event listener don't do anything right now, but I think it's conventional to have if we were working with multiple caches with different versions. I could remove these for right now since we're only using one cache?